IceCream/Classes/CKRecordRecoverable.swift

```diff
@@ -17,6 +17,45 @@ extension CKRecordRecoverable where Self: Object {
         let o = Self()
         for prop in o.objectSchema.properties {
             var recordValue: Any?
+            
+            if prop.isArray {
+                switch prop.type {
+                case .int:
+                    guard let value = record.value(forKey: prop.name) as? [Int] else { break }
+                    let list = List<Int>()
+                    list.append(objectsIn: value)
+                    recordValue = list
+                case .string:
+                    guard let value = record.value(forKey: prop.name) as? [String] else { break }
+                    let list = List<String>()
+                    list.append(objectsIn: value)
+                    recordValue = list
+                case .bool:
+                    guard let value = record.value(forKey: prop.name) as? [Bool] else { break }
+                    let list = List<Bool>()
+                    list.append(objectsIn: value)
+                    recordValue = list
+                case .float:
+                    guard let value = record.value(forKey: prop.name) as? [Float] else { break }
+                    let list = List<Float>()
+                    list.append(objectsIn: value)
+                    recordValue = list
+                case .double:
+                    guard let value = record.value(forKey: prop.name) as? [Double] else { break }
+                    let list = List<Double>()
+                    list.append(objectsIn: value)
+                    recordValue = list
+                case .data:
+                    guard let value = record.value(forKey: prop.name) as? [Data] else { break }
+                    let list = List<Data>()
+                    list.append(objectsIn: value)
+                    recordValue = list
+                default:
+                    break
+                }
+                break
```

Comment:
Hey, I don't get what you mean here? I haven't had any problems parsing from CKRecords.

Reply:
Yeah, I mean the `break` statement(Line 56) here will jump out of the for-in loop, so that the [L84](https://github.com/caiyue1993/IceCream/blob/c32c568132b31c83845ae4b0637acf9c477d7890/IceCream/Classes/CKRecordRecoverable.swift#L84) will not get executed.